### PR TITLE
chore(example): bump version of pluggy-sdk to 0.21.5 and fix webhook example.

### PR DESCRIPTION
**Pull request by NicolasMontone**

### CODE REVIEW

File: example/src/webhooks.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ void (async function(): Promise<void> {
   const webhooks = await client.fetchWebhooks()
   console.log(`Currently there are ${webhooks.results.length} webhook(s) configured.`)
 
-  let webhook = await client.createWebhook('all', WEBHOOK_URL)
+  let webhook = await client.createWebhook({ event: 'all', url: WEBHOOK_URL })
```
Comment (tmilar) [inline example/src/webhooks.ts:19]: Why This change of signature is only a patch version change? Shouldn't it had been at least minor?
I mean because version in package.json is changing from 0.21.0 to 0.21.5

Comment (NicolasMontone) [reply example/src/webhooks.ts:19]: You are right, I will merge this PR and force a new release with a minor change :)
